### ISSUE TITLE
Throw an error if GraknClient.cluster() does not receive a list

### ISSUE THREAD

**Issue by lolski** [graknlabs/client-python#209]
It is relatively common for a user to supply a single address string rather than a list of address. This should be validated early, and the error message should be user-readable.

**Comment by haikalpribadi** [comment]
Should this not be assigned to a project @lolski ? Production or Backlog?

**Comment by alexjpwalker** [comment]
This is resolved in 2.0.0 production, in which `Grakn.cluster_client` is capable of accepting either a single String or a list of Strings:
```py
@staticmethod
    def cluster_client(addresses: Union[Iterable[str], str], parallelisation: int = 2) -> GraknClusterClient:
        if isinstance(addresses, str):
            return _ClusterClient([addresses], parallelisation)
        else:
            return _ClusterClient(addresses, parallelisation)
```
